interactive/models.py

```diff
@@ -105,6 +105,9 @@ class RegistrationRequest(models.Model):
     organisation = models.CharField(max_length=100, verbose_name="Organisation")
     job_title = models.CharField(max_length=100, verbose_name="Job title")
 
+    is_approved = models.BooleanField(default=None, blank=True, null=True)
```

Comment:
I don't think we need an is_denied field if we use a nullable boolean.

Reply:
While we can do it this way (and it does avoid the need for is_denied), we're actually missing some information if we do.  We'll almost certainly want to know who approved or denied the registration for an audit trail (even if we don't have a formal one).

I think we have two options for that:

**Separate fields**
```
approved_at = …
approved_by = …

denied_at = …
denied_by = …

# a Constraint making sure each pair is always set together but both pairs are not set at the same time
```

**Combined fields with a status field**
```
reviewed_at = …
reviewed_by = …
review_status = models.TextField(choices=("Approved", "Denied"), null=True)

# a Constraint checking all three fields are either filled or not
```

I think the second one will use a simpler constraint so is probably easier to use.  Either solution can also add one or two properties to the model to expose the status without developers having to know the implementation details.